data/SAH_2022/metadata.yml

```diff
@@ -0,0 +1,451 @@
+source:
+  primary:
+    key: SAH_2022
+    bibtype: Online
+    year: 2022
+    author: State Herbarium of South Australia
+    title: eFlora of SA, Electronic Flora of South Australia
+    url: http://www.flora.sa.gov.au/
+contributors:
+  data_collectors:
+  - last_name: Kellermann
+    given_name: Jürgen
+    ORCID: 0000-0001-9124-9802
+    affiliation: State Herbarium of South Australia, Botanic Gardens and State Herbarium,
+      Hackney Road, Adelaide, SA 5000, Australia
+  - last_name: Coleman
+    given_name: David
+    ORCID: 0000-0002-1773-6597
+    affiliation: Evolution & Ecology Research Centre, School of Biological, Earth,
+      and Environmental Sciences, UNSW Sydney, Australia
+    notes: Responsible for extracting data from online resource
+    additional_role: contact
+  austraits_curators: Elizabeth Wenk
+dataset:
+  data_is_long_format: no
+  custom_R_code: '
+    data %>%
+      filter(str_detect(taxon_name, " ")) %>%
```

Comment:
definitely needed?

Reply:
Yes. To filter out genus and family-level values since AusTraits is only meant to include traits at the species-level.